data_and_models/pipelines/ner/dvc.lock

```diff
@@ -12,12 +12,12 @@ stages:
       md5: 117da032ef2e2792429dff88c06c90b7
       size: 62653
     - path: ../../models/ner_er/model1
-      md5: 12bfb6ab6750bb670005b4722ff05104.dir
-      size: 171634265
+      md5: 189c70c7e75a4c9c83c986d03460cc23.dir
+      size: 171634305
       nfiles: 21
     - path: Dockerfile
-      md5: 9a4e8a0fffb0a7e49faa347f2ec2132d
-      size: 1732
+      md5: 7863c45504f96d4d5147ae07f8918759
+      size: 1772
```

Comment:
1. It looks like all the  `ner/modelX/model-best/` and all the `ner/modelX/model-best/` have changed. Do you have an idea why?
2. On the contrary, none of the `../../metrics/ner/xxx.json` seems to have changed. How does this relates to the previous point? By the way, if some metric had changed the `xxx.json` should have also been `git add`ed to this PR right?
3.  The `Dockerfile` was not touched in this PR, is it normal that its `md5` has changed?

Reply:
1. Yes it is. Each `ner/modelX/model-best/` folders contains a copy of the `config.cfg`. From #347, we needed to change the initialization directly in that file. This is the reason why the `md5` changed. In terms of implementation, there is no change in the logic (you can compare the [spacy commit](https://github.com/explosion/spaCy/commit/bdb485cc80ca6822471eac0a29ac2782d9b55450) and our previous [initialize.py](https://github.com/BlueBrain/Search/blob/master/data_and_models/pipelines/ner/initialize.py) )
2. The `md5` of the model has changed but the weights did not. Side note concerning this, we recently changed the implementation of our `eval.py` but the version of BBS pinned in the `Dockerfile` is still `v0.1.2`. Once this pinned version is going to change, the `metrics/ner/xxx.json` are going to be affected. Those files are tracked with `git`, it is going to be needed to be add them once it is done.
3. `Dockerfile` was touched in [previous commit](https://github.com/BlueBrain/Search/commit/b9c886966ca4d893b41457a17262e198e3ba7f03) but the `md5` was not updated at that time. Personally, I don't think it is a problem we don't update `dvc.lock` each time but I can understand it is confusing!